### ISSUE TITLE
No way to add an existing author to an article

### ISSUE THREAD

**Issue by jobiddulph** [dadi/publish#748]
When in an article, I would expect to be able to select an author and add it to the article that I am currently updating

What actually happens is that I click on 'Select an existing author', it takes me to the list of authors, I click on the name of one to add it to my article and this takes me to the author content page, with no way to add it to the article. If I click save and continue I've just updated the author content page

Create a new article add title, excerpt and body copy, click on 'Select existing author' and as above click on an author name

Publish v.3.0.0

**Comment by eduardoboucas** [comment]
@jobiddulph This happens when you click on the name of the author, right? If you click anywhere else on the row, or on the checkbox, it selects the item and then allows you to save. Can you please check?

In any case, this is definitely a UX problem that we should revisit.

**Comment by jobiddulph** [comment]
@eduardoboucas yes that is the case, I've tweaked the title of the ticket

**Comment by eduardoboucas** [comment]
Thank you. I've added the `status: needs UI` label, as this is something that should be part of our upcoming UX review.

**Comment by eduardoboucas** [comment]
TL;DR for @josephdenne: when you're on the reference select view, you can click anywhere in a row to select it, expect in the value of the primary field (in [this case](http://develop.publish.dadi.technology/articles/222b7b3d-3b14-4ddc-ae25-008df3c63864/select/author), the name of the author), which takes you to the edit view of the referenced document. This has proven to be confusing, because the primary action should be selecting a reference value, not navigating to the referenced document.

Maybe we could remove the link from the primary field and instead add a small icon that takes people to the referenced document? Something like:

![download](https://user-images.githubusercontent.com/4162329/62454562-76438c00-b76c-11e9-8fc3-81aeec2d5eca.png)
